File: SeleniumDemo/src/test/com/sitename/selenium/base/SeleniumTestAbstract.java
package test.com.sitename.selenium.base;


import org.junit.After;
import org.junit.Before;

import test.com.sitename.selenium.util.Selenium_Util;


public class SeleniumTestAbstract {
	@Before
	public void setUpBeforeClass() {
		Selenium_Util.startWebDriver();
//		Selenium_Util.maximize();
		Selenium_Util.implicitlyWait();
//		Selenium_Util.pageLoadTimeout();
		
	}
    @After
	public void tearDownAfterClass() {
		Selenium_Util.quit();
    	
	}
}

File: PythonDemo/src/test/com/sitename/selenium/base/SeleniumTestAbstract.py
'''
Created on 2012-12-23

@author: Administrator
'''
import unittest
from test.com.sitename.selenium.util.Selenium_Util import Selenium_Util


class SeleniumTestAbstract(unittest.TestCase):
    def setUp(self):
        Selenium_Util.startBowser()
        Selenium_Util.implicityWait()
    
    def tearDown(self):
        Selenium_Util.quit()

